shared/analytics/mixpanelUtil.ts

```diff
@@ -0,0 +1,20 @@
+import { MixpanelPayload } from './types';
```

Comment:
Any reason we need a separate mixpanel file for server/browser? That kinda defeats the purpose of putting it in `shared/`.